src/doom/am_map.c

```diff
@@ -1545,20 +1545,21 @@ void AM_drawWalls(void)
 	    {
 		// [crispy] draw keyed doors in their respective colors
 		// (no Boom multiple keys)
+        // make keyed doors flash for easier visibility
```

Comment:
check indentation here

Reply:
It's because I used spaces and the other comments use tabs. They line up when tabs are set to four spaces.

Reply:
Please don't introduce a new indentation scheme.